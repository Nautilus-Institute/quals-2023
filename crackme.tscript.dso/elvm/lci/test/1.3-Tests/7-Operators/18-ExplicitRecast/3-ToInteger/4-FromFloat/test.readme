This test checks that the explicit recast operator correctly casts floating
point decimal values to integer types.
