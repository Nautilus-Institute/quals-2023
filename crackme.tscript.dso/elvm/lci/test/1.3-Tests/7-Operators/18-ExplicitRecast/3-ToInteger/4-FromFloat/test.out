0
1
4
-4
